modules/exploits/windows/http/sharepoint_workflows_xoml.rb

```diff
@@ -70,7 +70,7 @@ def check
     return CheckCode::Unknown('Did not receive an HTTP 200 OK response') unless res&.code == 200
 
     compiler_errors = extract_compiler_errors(res)
-    return CheckCode::Unknown('No compiler errors were reported') unless compiler_errors&.length > 0
+    return CheckCode::Unknown('No compiler errors were reported') unless compiler_errors&.length.& > 0
```

Comment:
Uhhhh....what happened here? Not sure why this adjustment was made, am I missing something here?

Reply:
On master:

```
$ rubocop modules/exploits/windows/http/sharepoint_workflows_xoml.rb
Inspecting 1 file
W

Offenses:

modules/exploits/windows/http/sharepoint_workflows_xoml.rb:73:97: W: Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.
    return CheckCode::Unknown('No compiler errors were reported') unless compiler_errors&.length > 0
                                                                                                ^^^^
```

https://docs.rubocop.org/rubocop/1.0/cops_lint.html#lintsafenavigationchain

Example:

```
2.7.2 :001 > []&.length > 0
 => false 
2.7.2 :002 > nil&.length > 0
Traceback (most recent call last):
    ....
(irb):2:in `<main>': undefined method `>' for nil:NilClass (NoMethodError)
```

Fix:
```
2.7.2 :003 > []&.length&.> 0
 => false 
2.7.2 :004 > nil&.length&.> 0
 => nil 
```

In this case the falsey value is fine

Reply:
See https://github.com/rapid7/metasploit-framework/pull/14806#discussion_r582862948 for an update on this, there was a typo in the proposed fix which has now been addressed. Marking resolved.